test_article_init failing: tags don't match

```
$ nosetests
.F..
======================================================================
FAIL: Ensure class parse method gets all desired fields.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/paregorios/Envs/isaw.awol/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/paregorios/Documents/files/I/isaw.awol/isaw/awol/tests/test_awol_article.py", line 45, in test_article_init
    assert_equals(r.keywords, [u'Journal', u'Open Access'])
AssertionError: Lists differ: [u'antiquity', u'archaeology',... != [u'Journal', u'Open Access']

First differing element 0:
antiquity
Journal

First list contains 9 additional elements.
First extra element 2:
art

+ [u'Journal', u'Open Access']
- [u'antiquity',
-  u'archaeology',
-  u'art',
-  u'cultural heritage',
-  u'culture',
-  u'heritage',
-  u'history',
-  u'journal',
-  u'law',
-  u'museums',
-  u'open access']
-------------------- >> begin captured logging << --------------------
langid.langid: INFO: initializing identifier
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 4 tests in 1.238s

FAILED (failures=1)
(isaw.awol)
```

--- a/isaw/awol/tests/test_awol_article.py
+++ b/isaw/awol/tests/test_awol_article.py
@@ -28,7 +28,7 @@ def teardown_function():
     pass
 
 @with_setup(setup_function, teardown_function)
-def test_article_init():
+def test_awol_article_init():
     """Ensure class parse method gets all desired fields."""
 
     file_name = os.path.join(PATH_TEST_DATA, 'post-capitale-culturale.xml')
@@ -42,7 +42,7 @@ def test_article_init():
     assert_equals(r.domain, 'www.unimc.it')
     assert_is_instance(r.subordinate_resources, list)
     assert_is_instance(r.keywords, list)
-    assert_equals(r.keywords, [u'Journal', u'Open Access'])
+    assert_equals(r.keywords, [u'antiquity', u'archaeology', u'art', u'cultural heritage', u'culture', u'heritage', u'history', u'journal', u'law', u'museums', u'open access'])
     assert_equals(r.language, ('it', 1.0))
     assert_is_instance(r.related_resources, list)
     assert_equals(r.title, 'Il capitale culturale')

